packages/embeds/embed-react/test-cal.tsx

```diff
@@ -1,5 +1,4 @@
-import { useEffect } from "react";
-import { useState } from "react";
+import React, { useEffect, useState } from "react";
```

Comment:
got the standard "react is not part of umd" error here - hence the added `React` import